handbook/demand/README.md

```diff
@@ -116,6 +116,14 @@ Any changes to the current running ads visible to a user, including designs, key
 ### Engage with the community
 Public conversations on social media create valuable opportunities for contributors to answer technical questions and collect feedback.
 
+### Promote a post on LinkedIn
+Original posts on LinkedIn by Fleet employees can be promoted using the Fleet company page. If you think your post would make sense in front of a bigger audience, [create an issue on the demand board](https://fleetdm.com/handbook/demand#contact-us).
+
+1. Create the post on your personal LinkedIn profile. (Only original posts. Reposts and quotes of other posts can be promoted)
```

Comment:
This should be written as your training a Demand team member. Lets combine this and give it a bigger stage!